charts/pulsar/values.yaml

```diff
@@ -652,6 +652,9 @@ pulsar_metadata:
   # configurationStore:
   configurationStoreMetadataPrefix: ""
   configurationStorePort: 2181
+  ## optional you can specify a nodeSelector for all init jobs (pulsar-init & bookkeeper-init)
+  # nodeSelector:
+    # cloud.google.com/gke-nodepool: default-pool
```

Comment:
Nit: I think it is valuable to add the key to the values file.

```suggestion
  nodeSelector: {}
```

Reply:
This is a larger style issue that can be addressed later. Going to merge this PR without worrying about this fix.